Pod/Classes/ResizingTextView.swift

```diff
@@ -86,29 +96,36 @@ open class ResizingTextView: UITextView {
         NotificationCenter.default.removeObserver(self)
     }
     
+    open override func layoutSubviews() {
+        super.layoutSubviews()
+        didUpdateText(self)
```

Comment:
This is great for handling the initial size, however, if the user had their text view to have a LARGER initial size than one line (for example in their nib), this could shrink the text view on initial layout.

If we are adding in a maximum size, we should really just add a minimum size also. That would fix this issue by making it explicit that this is going to resize on initial load if needed to be within the max/min range.

Minimum size should default to `.numberOfLines(1)`